sycl/test-e2e/Matrix/SG32/joint_matrix_bfloat16_rowmajorA_rowmajorB.cpp

```diff
@@ -5,8 +5,10 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// REQUIRES: matrix
+// REQUIRES: aspect-ext_intel_matrix
 // REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
+// VNNI transform is not supported yet on DG2
+// UNSUPPORTED: gpu-intel-dg2
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
```

Comment:
Is the comment below and XFAIL still valid?

Reply:
Yes, SG32 VNNI is still failing on PVC

Reply:
> // This tests support of row major layout for matrix B which does automatic VNNI
> // transform. This is currently only available on AMX
This comment doesn't mention sub-group size 32 VNNI. It sounds like VNNI is only available on AMX.

I would suggest updating the comment as follows:
// This tests support of row major layout for matrix B which does automatic VNNI transform.
Should go somewhere to the top as general test description

// Sub-group size 32 support for this test is not currently available in IGC
should go right before //XFAIL:gpu